spring-cloud-gcp-data-datastore/src/main/java/org/springframework/cloud/gcp/data/datastore/core/convert/ReadWriteConversions.java

```diff
@@ -51,10 +53,22 @@
 	 * {@code null} if the property is a singular object.
 	 * @param targetComponentType the type of the property to convert. For collection-like
 	 * properties this refers to the individual items' type.
+	 * @param isEmbeddedComponents {@code true} if the property is collection-like and its
+	 * items are individually stored as embedded entities.
 	 * @return an object of a target type.
 	 */
 	<T> T convertOnReadEmbedded(Object val, Class targetCollectionType,
-			Class targetComponentType);
+			Class targetComponentType, boolean isEmbeddedComponents);
```

Comment:
"Embedded" might cause some to think it is the JPA/sql style embedded.  We should rename the annotation to "EmbeddedEntity" to make that clear.

Reply:
no longer necessary since i am now using the entity annotation on the embedded object's class.